core-processor/src/main/java/io/micronaut/inject/beans/visitor/BeanIntrospectionWriter.java

```diff
@@ -594,6 +608,14 @@ private void writeIntrospectionClass(ClassWriterOutputVisitor classWriterOutputV
         }
     }
 
+    private void writeBooleanMethod(ClassWriter classWriter, String methodName, boolean isBuildable) {
+        GeneratorAdapter isBuildableMethod = startPublicMethodZeroArgs(classWriter, boolean.class, methodName);
+        isBuildableMethod.push(isBuildable);
+        isBuildableMethod.returnValue();
+        isBuildableMethod.visitMaxs(2, 1);
+        isBuildableMethod.endMethod();
+    }
+
```

Comment:
Rename the argument from `isBuildable`:
```suggestion
    private void writeBooleanMethod(ClassWriter classWriter, String methodName, boolean returnValue) {
        GeneratorAdapter booleanMethod = startPublicMethodZeroArgs(classWriter, boolean.class, methodName);
        booleanMethod.push(returnValue);
        booleanMethod.returnValue();
        booleanMethod.visitMaxs(2, 1);
        booleanMethod.endMethod();
    }

```